src/angularWhitespaceRule.ts

```diff
@@ -162,8 +162,16 @@ class PipeWhitespaceVisitor extends RecursiveAngularExpressionVisitor implements
 
 
     const replacements = [];
+    let parentheses = false;
+    let leftBeginning: number;
+    if(sf[exprEnd] === ')') {
```

Comment:
Can we have the parenthesis surrounded by whitespace?

Reply:
yes, we can. The rule doesn't check that.